lib/PuppeteerSharp.Tests/Page/OfflineModeTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Net;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace PuppeteerSharp.Tests.Page
+{
+    [Collection("PuppeteerLoaderFixture collection")]
+    public class OfflineModeTests : PuppeteerPageBaseTest
+    {
+        [Fact]
+        public async Task ShouldWork()
+        {
+            await Page.SetOfflineModeAsync(true);
+            await Assert.ThrowsAsync<NavigationException>(async () => await Page.GoToAsync(TestConstants.EmptyPage));
+
+            await Page.SetOfflineModeAsync(false);
+            var response = await Page.ReloadAsync();
+            Assert.Equal(HttpStatusCode.OK, response.Status);
+        }
+
+        [Fact]
+        public async Task ShouldEmulateNavigatorOnLine()
+        {
+            const string navigatorOnLine = "window.navigator.onLine";
```

Comment:
I know this is used 3 times in the test but it's a bit less clear this way, might be clearer to use that string on each `EvaluateExpressionAsync` call

Reply:
Makes sense

Reply:
Done